Tasks/Xcode/Tests/L0.ts

```diff
@@ -338,16 +264,14 @@ describe('Xcode L0 Suite', function () {
             'OBJROOT=/user/build/output/$(SDK)/$(Configuration)/build.obj ' +
             'SYMROOT=/user/build/output/$(SDK)/$(Configuration)/build.sym ' +
             'SHARED_PRECOMPS_DIR=/user/build/output/$(SDK)/$(Configuration)/build.pch ' +
-            'OTHER_CODE_SIGN_FLAGS=--keychain=/user/build/_xcodetasktmp.keychain ' +
```

Comment:
This will be useful to set to APPLE_CERTIFICATE_KEYCHAIN/equivalent output variable if available. The advantage is if we pass this, Xcode will not have to look in all the keychains in the search path, it will only look in this keychain. It could be potentially faster and avoid conflicts or accidentally pick another certificate.

Reply:
Understood. Let's save this for another PR. Several of us can discuss whether adding a Keychain name variable to the Xcode task is an improvement or confusing.